packages/platform-sdk/test/helpers/test-utils.ts

```diff
@@ -9,7 +9,9 @@ export function requireEnvVar(varName: string): string {
 export async function waitUntil(
   waitCondition: () => Promise<boolean>,
   maxRetry: number = 10,
-  maxWaitingTimePerRetryInMs: number = 32000
+  maxWaitingTimePerRetryInMs: number = process.version.startsWith('v22')
```

Comment:
better increase it in general than a version dependent increase. Timeout is not related to the node version